index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Youssef's Portfolio</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="header" id="container">
+      <a href="#" class="logo">Youssef</a>
+      <nav class="navbar">
+        <ul>
+          <li><a href="#container">Home</a></li>
+          <li><a href="#about">About me</a></li>
+          <li><a href="#projects">Projects</a></li>
+          <li><a href="#contact">Contact me</a></li>
+        </ul>
+      </nav>
+    </div>
+
+    <!-- main---------------------- -->
+
+    <div class="main" id="container">
```

Comment:
I would not recommend to use main as classname, as you could have `main` tag, and in general in can be confusing.